scanners/shell-history.sh

```diff
@@ -49,6 +49,7 @@ shell_history_blacklist[35]="193.187.174.104"
 shell_history_blacklist[36]="62.113.112.33"
 shell_history_blacklist[37]="217.12.221.12"
 
+shell_history_blacklist[37]="5.101.0.209"
```

Comment:
shell_history_blacklist[38]="5.101.0.209"

Reply:
is this an IP that is seen scanning, or has been used after exploitation?

Reply:
also, the array index should be `38`

Reply:
Yes, it has been spotted scanning.
 I corrected it in the comment
check the link below 
https://www.abuseipdb.com/check/5.101.0.209